modules/cnf-scheduling-exact-based-on-reource.adoc

```diff
@@ -0,0 +1,122 @@
+
+If you want to change where the workloads run you can add the `scoringStrategy` to the `NUMAResourcesScheduler` custom resource and set it`s value to either `MostAllocated`  or `BalancedAllocation`. 
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Delete the currently running `NUMAResourcesScheduler` resource:
+
+.. Get the active `NUMAResourcesScheduler` by running the following command:
++
+[source,terminal]
+----
+$ oc get NUMAResourcesScheduler
+----
++
+.Example output
+[source,terminal]
+----
+NAME                     AGE
+numaresourcesscheduler   92m
+----
+
+.. Delete the secondary scheduler resource by running the following command:
++
+[source,terminal]
+----
+$ oc delete NUMAResourcesScheduler numaresourcesscheduler
+----
++
+.Example output
+[source,terminal]
+----
+numaresourcesscheduler.nodetopology.openshift.io "numaresourcesscheduler" deleted
+----
+
+. Save the following YAML in the file `nro-scheduler-mostallocated.yaml`. This example changes the `scoringStrategy` to `MostAllocated`:
++
+[source,yaml]
+----
+apiVersion: nodetopology.openshift.io/v1
+kind: NUMAResourcesScheduler
+metadata:
+  name: numaresourcesscheduler
+spec:
+  imageSpec: "registry.redhat.io/openshift4/noderesourcetopology-scheduler-container-rhel8:v{product-version}"
+  scoringStrategy:
+        type: "MostAllocated" <1>
+----
++
+<1> If the `scoringStrategy` configuration is omitted the default of `LeastAllocated` applies.
+
+. Create the updated `NUMAResourcesScheduler` resource by running the following command:
++
+[source,terminal]
+----
+$ oc create -f nro-scheduler-mostallocated.yaml
+----
++
+.Example output
+[source,terminal]
+----
+numaresourcesscheduler.nodetopology.openshift.io/numaresourcesscheduler created
+----
+
+.Verification steps
+
+. Check that the NUMA-aware scheduler was successfully deployed:
+
+.. Run the following command to check that the CRD is created successfully:
++
+[source,terminal]
+----
+$ oc get crd | grep numaresourcesschedulers
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                                              CREATED AT
+numaresourcesschedulers.nodetopology.openshift.io                 2022-02-25T11:57:03Z
+----
+
+.. Check that the new custom scheduler is available by running the following command:
++
+[source,terminal]
+----
+$ oc get numaresourcesschedulers.nodetopology.openshift.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                     AGE
+numaresourcesscheduler   3h26m
+----
+
+. Verify that the `ScoringStrategy` has been applied correctly, by checking the relevant `ConfigMap` for the scheduler by running the following command: 
```

Comment:
Drop comma / add "resource":

```suggestion
. Verify that the `ScoringStrategy` has been applied correctly by checking the relevant `ConfigMap` resource for the scheduler by running the following command: 
```
or maybe a slight rewrite to avoid multiple "by"s:

```suggestion
. Verify that the `ScoringStrategy` has been applied correctly by running the following command to check the relevant `ConfigMap` resource for the scheduler:
```